templates/ssl_ratios_v1.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-derived-datasets.telemetry.ssl_ratios_v1` AS
+WITH windowed AS (
+  SELECT
+    *,
+    SUM(ssl_loads) OVER w1 + SUM(non_ssl_loads) OVER w1 AS total_loads
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.ssl_ratios_raw_v1`
+  WINDOW
+    w1 AS (ORDER BY submission_date ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING))
+SELECT
+  submission_date,
+  os,
+  country,
+  -- ratio of pings that have the probe
+  reporting_ratio,
+  -- normalized count of pageloads that went into this ratio
+  (non_ssl_loads + ssl_loads) / total_loads AS normalized_pageloads,
+  ssl_loads / (non_ssl_loads + ssl_loads) AS ratio
```

Comment:
note that `/` in BigQuery does not do integer division when given two integers (unlike Athena), so these don't need a `1.0 *`